pytorch/inference/docker/1.10/py3/sdk1.17.1/Dockerfile.neuron

```diff
@@ -60,28 +61,26 @@ RUN keytool -importkeystore -srckeystore /etc/ssl/certs/java/cacerts -destkeysto
     mv /etc/ssl/certs/java/cacerts.jks /etc/ssl/certs/java/cacerts; \
     /var/lib/dpkg/info/ca-certificates-java.postinst configure;
 
-RUN curl -L -o ~/miniconda.sh https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh \
- && chmod +x ~/miniconda.sh \
- && ~/miniconda.sh -b -p /opt/conda \
- && rm ~/miniconda.sh \
+RUN curl -L -o ~/mambaforge.sh https://github.com/conda-forge/miniforge/releases/download/${MAMBA_VERSION}/Mambaforge-${MAMBA_VERSION}-Linux-x86_64.sh \
+ && chmod +x ~/mambaforge.sh \
+ && ~/mambaforge.sh -b -p /opt/conda \
+ && rm ~/mambaforge.sh \
  && /opt/conda/bin/conda update conda \
- && /opt/conda/bin/conda install -c conda-forge \
+ && /opt/conda/bin/conda install -c conda-forge -y \
     python=$PYTHON_VERSION \
- && /opt/conda/bin/conda install -y \
-    # conda 4.10.0 requires ruamel_yaml to be installed. Currently pinned at latest.
-    ruamel_yaml==0.15.100 \
```

Comment:
Why did we remove `ruamel_yaml`?

Reply:
I believe we have done this on all the PyTorch dockerfiles where we moved to using mamba instead of miniconda.

Example:
https://github.com/aws/deep-learning-containers/commit/d7f156a0108b8af216fffd49ee2d47e4031c1adb